breakdown_test.go

```diff
@@ -510,7 +510,7 @@ func spanSelfTimeMetrics(txName, txType, spanType, spanSubtype string, count int
 		},
 		Samples: map[string]model.Metric{
 			"span.self_time.count":  {Value: float64(count)},
-			"span.self_time.sum.us": {Value: sum.Seconds() * 1000000},
+			"span.self_time.sum.us": {Value: float64(sum) / 1e3},
```

Comment:
I had to change this since there was a case where `510000` turned into `510.00000000000006` due to the `time.Duration.Seconds()` logic. In any case, this should be the same as the other operation, perhaps a little more efficient, since we're only doing 1 division.